examples/uidl-samples/project.json

```diff
@@ -526,15 +547,22 @@
             "width": "300px",
             "border": "1px solid #ccc",
             "padding": "10px",
-            "margin": "5px"
+            "margin": "5px",
+            "color": "inherit"
           },
           "children": [
             {
               "type": "element",
               "content": {
                 "elementType": "image",
-                "attrs": {
-                  "url": "$props.url"
+                "abilities": {
+                  "link": {
+                    "type": "url",
+                    "options": {
+                      "url": "$props.link",
```

Comment:
Update validator to check for the url and link in abilities if they are being used from props. Right now if we run it throws validation errors for `link` and `url` even if they are being used. This component should throw only `url` from `propDefinitions` in validations.

Reply:
I added link.options.url in the traverse nodes function, it should be passed now when the validator is looking at all usages of UIDLAttributes. can you check again?